src/marshaller/HipieDDL.js

```diff
@@ -1287,7 +1319,7 @@
         var datasourceRequestOptimizer = new DatasourceRequestOptimizer();
         for (var key in this.visualizationRequests) {
             var item = this.visualizationRequests[key];
-            if (item.updateVisualization.type !== "GRAPH") {
+            if (!this.skipClear && item.updateVisualization.type !== "GRAPH") {
```

Comment:
PrimeData will should be clearing the visualization here:  https://github.com/GordonSmith/Visualization/blob/c48e2463ad5d0cde256af0c79e6d6f16e18ec873/src/marshaller/HipieDDL.js#L1553-L1565